meteor/client/ui/RundownView/RundownTiming/withTiming.tsx

```diff
@@ -102,12 +119,57 @@ export function withTiming<IProps, IState>(
 				// To bring it back to sync, we mark the component as dirty, which will
 				// force an update on the next low resoluton tick, regardless of what
 				// the filter says.
-				if (!!this.filterGetter && durations.isLowResolution !== !expandedOptions.isHighResolution) {
+				if (componentIsDirty(this.filterGetter, highResDurations, expandedOptions.dataResolution)) {
 					this.isDirty = true
 				}
 
-				return <WrappedComponent {...this.props} timingDurations={durations} />
+				return (
+					<WrappedComponent
+						{...this.props}
+						timingDurations={rundownTimingDataFromDataResolution(expandedOptions.dataResolution, {
```

Comment:
This will create an object on every update and then Garbage-Collect the old inline object. The point of doing `timingDurations={durations}` was to remove that garbage being created so that there's nothing to GC (because we're just passing references).